sklearn/cluster/_birch.py

```diff
@@ -134,7 +134,7 @@ class _CFNode:
         view of ``init_sq_norm_``.
 
     """
-    def __init__(self, threshold, branching_factor, is_leaf, n_features):
```

Comment:
Tests failing because internally we use,
```py
_CFNode(threshold, branching_factor, is_leaf=True,
        n_features=n_features)
```
I'm not convinced that,
```py
_CFNode(threshold=threshold, branching_factor=branching_factor, is_leaf=True,
        n_features=n_features)
```
is an improvement. It's a private class anyway.

Reply:
I agree it may not look really neat, but it does remove the possibility of some potential bugs.